kivy/uix/gridlayout.py

```diff
@@ -295,9 +322,18 @@ def _init_rows_cols_sizes(self, count):
             Logger.warning('%r have no cols or rows set, '
                            'layout is not triggered.' % self)
             return
+
         if current_cols is None:
+            if self._fills_row_first:
+                Logger.warning(
```

Comment:
I'm just taking a quick skim, and that should be a error, not a warning. We should almost never have warnings in code. This is an error in the configuration so a error should be raised.

Reply:
ok, got it.